src/App.css

```diff
@@ -1,5 +1,9 @@
-.App {
+.App > div {
```

Comment:
Why > div here?

Reply:
Ah I could give each parent div element the same class and just style that for example. So I don't have to use the class "App" for the general grid layout, which should be the same on each page, at all🤔